pages/explorer/races/[raceId].tsx

```diff
@@ -59,11 +60,20 @@ const RaceDetail: NextPage<RaceDetailProps> = ({
   const raceToFetch = (query.raceId as raceType) || "american";
   const localizedRace = localizedNames[raceToFetch];
 
+  const metaKeywords = generateKeywordsString([
+    `${localizedRace}`,
+    `${localizedRace} explorer`,
+    `${localizedRace} battle groups`,
+    `${localizedRace} units`,
+  ]);
+
   return (
     <>
       <Head>
-        <title>{localizedRace} - COH3 Explorer</title>
+        <title>{`${localizedRace} - COH3 Explorer`}</title>
```

Comment:
When passing into <HEAD> elements, it's important to have it as "single" element, or it might do problems.